src/overlays/actors/ovl_En_Kakasi/z_en_kakasi.c

```diff
@@ -46,80 +77,1172 @@ const ActorInit En_Kakasi_InitVars = {
     (ActorFunc)EnKakasi_Draw,
 };
 
-#endif
+Vec3f D_80971DCC[] = {
```

Comment:
```suggestion
static Vec3f D_80971DCC[] = {
```
Make sure all your data is prefixed with static, applies to the data below